<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main</class>
 <widget class="QMainWindow" name="frm_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_plagiatChecker">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plagiat Checker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ergebnisse">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>1201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ergebnisse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tv_ergebnisse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1181</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="te_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1181</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_ffnen">
   <property name="text">
    <string>Ã–ffnen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_check</sender>
   <signal>clicked()</signal>
   <receiver>bt_check</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
